pages/questionpage/questionpage.css

```diff
@@ -0,0 +1,45 @@
+.question-section {
+	margin: 2rem 12rem;
+}
+.question-section div {
+	margin: 1rem 0;
+}
+
+.question-section div .btn-outline-secondary:hover {
+	background-color: transparent;
+	cursor: default;
+	color: var(--colour-primary);
```

Comment:
Maybe you can avoid this cursor: default. As it is set to default anyways.

Reply:
This is to override the default.